Library/Homebrew/utils/backtrace.rb

```diff
@@ -0,0 +1,25 @@
+# typed: true
+# frozen_string_literal: true
+
+module Utils
+  module Backtrace
+    # Cleans `sorbet-runtime` gem paths from the backtrace unless...
+    # 1. `verbose` is set
+    # 2. first backtrace line starts with `sorbet-runtime`
+    #   - This implies that the error is related to Sorbet.
+    sig { params(error: Exception).returns(T.nilable(T::Array[String])) }
+    def self.clean(error)
+      backtrace = error.backtrace
+
+      return backtrace if Context.current.verbose?
```

Comment:
Not blocking this being merged but: this feels weird to me as I don't think I'd expect `--verbose` to modify backtraces.

Reply:
> Not blocking this being merged but: this feels weird to me as I don't think I'd expect `--verbose` to modify backtraces.

Fair point! But under what circumstances does one expect backtraces to be modified? (I like your suggestion below to echo that sorbet lines are removed.)

Reply:
The point here is that using `--verbose` leave backtraces unmodified. My thinking is that you'd expect more output with `--verbose` and that means in this case that backtraces might be longer.